mobile/android/java/Interface.java

```diff
@@ -8,4 +8,5 @@
     public static native String iota_sign_signature_gen_trytes(String seed, int index, int security, String bundleHash);
     public static native byte[] iota_sign_signature_gen_trits(byte[] seed, int index, int security, byte[] bundleHash);
     public static native String iota_digest(String trytes);
+    public static native String bundle_miner_mine(byte[] bundle_normalized_max, int security, byte[] essence, int essence_length, int count, int nprocs);
```

Comment:
```suggestion
    public static native long bundle_miner_mine(byte[] bundle_normalized_max, int security, byte[] essence, int essence_length, int count, int nprocs);
```
Should this be `long` since the return type in [Interface.h](https://github.com/iotaledger/entangled/pull/1445/files#diff-d3b7c9101f69c7d0205e2b1df1bcda72R71) is `jlong`?

Reply:
Yes, definitely a failed copy/paste, thanks!